### PR TITLE
Fix wrong PHPDocs in \Pimcore\Model\Document\Tag\Image

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

File: models/Document/Tag/Image.php [hunk 1]
```diff
@@ -421,7 +423,7 @@ public function getImage()
     }
 
     /**
-     * @param Asset\Image $image
+     * @param Asset\Image|null $image
```
Comment (NiklasBr) [inline models/Document/Tag/Image.php:426]: `$this` cannot be `null` right?

Comment (blankse) [reply models/Document/Tag/Image.php:426]: The image param can be null. To reset the tag. See:
https://github.com/pimcore/pimcore/pull/7020/files#diff-3bbec05fe0a61dca677cb7e8c81411dfR803

Comment (NiklasBr) [reply models/Document/Tag/Image.php:426]: But the method returns `$this`

Comment (blankse) [reply models/Document/Tag/Image.php:426]: Yes `$this` cannot be null. And the return PHPDoc is correct, right?

Comment (NiklasBr) [reply models/Document/Tag/Image.php:426]: says `@return Model\Document\Tag\Image|null` now

Comment (jdreesen) [reply models/Document/Tag/Image.php:426]: No, it doesn't @blankse only changed the `@param` annotation, not the `@return` one.

Comment (NiklasBr) [reply models/Document/Tag/Image.php:426]: image

Incognito says I was wrong, might have seen at an old commit or something. Sorry!

**Comment by brusch** [comment]
Thanks!

**Pull request merged**